crates/interpreter/src/instructions/host.rs

```diff
@@ -368,6 +368,9 @@ pub fn static_call<H: Host, SPEC: Spec>(interpreter: &mut Interpreter<'_>, host:
 
 // EIP-3074
 pub fn auth<H: Host, SPEC: Spec>(interpreter: &mut Interpreter<'_>, host: &mut H) {
+    // The minimum amount of memory expansion (in bytes) required to hold the signature and commit.
+    const MINIMUM_EXPANSION: usize = 97;
```

Comment:
I don't believe this is necessarily true. It's possible to specify a memory region less than 97 bytes and the eip says to treat the rest of the expected bytes as zeros.